go.mod

```diff
@@ -1,9 +1,12 @@
 module github.com/anz-bank/go-course
 
 require (
+	bou.ke/monkey v1.0.1
```

Comment:
According to the other conversation with @juliaogris, you were able to get rid of this, does this dependency still need to be here (I couldn't find an import, but maybe I'm just blind)

Reply:
this is autogenerated and refers to `lab10` which is already in the master.